test/e2e/common_test.go

```diff
@@ -40,13 +44,32 @@ type PodmanTestIntegration struct {
 	SignaturePolicyPath string
 	CgroupManager       string
 	Host                HostOS
+	Timings             []string
 }
 
+var LockTmpDir string
+
 // PodmanSessionIntegration sturct for command line session
 type PodmanSessionIntegration struct {
 	*PodmanSession
 }
 
+type testResult struct {
+	name   string
+	length float64
```

Comment:
If you tracked pass/fail/flake status here as well, and reported on it, it might help with debugging future less-obvious performance problems.  For example, maybe a test only performs badly when it's failing, but a subsequent flake re-run passes.  Just an idea.

Reply:
yeah, lets add more in the next go around!